projects/demo/src/modules/components/loader/loader.component.ts

```diff
@@ -44,6 +48,11 @@ export class ExampleTuiLoaderComponent {
         HTML: example3Html,
     };
 
+    readonly example4: FrontEndExample = {
```

Comment:
Please change all raw imports to inline `import` statements like in this file:
https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo/src/modules/components/combo-box/combo-box.component.ts

We try to migrate all demo portal to this new shorter format 🙂

Reply:
done ✅

Reply:
Thanks the for review guys, i have updated the PR. have a look :)